### PR TITLE
Fixes for symfony 5 support

### PR DESCRIPTION
**Pull request by acrobat**
| Q             | A
| ------------- | ---
| Branch?       | "symfony-5" 
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | 
| License       | MIT
| Doc PR        | reference to the documentation PR, if any

@dbu I did some changes to make (part of) the build pass. Same changes like we did for the routing component.

- Dropped php  7.1 
- Dropped sf 3.4 and 4.3 support

Only the functional tests still fail, but I don't know what we should do. Are these missing dependencies from he testing package or should we add them here as dev dependencies?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,23 +15,23 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "symfony-cmf/routing": "^2.1.0@dev",
-        "symfony/framework-bundle": "^3.4 || ^4.3 || ^5.0",
+        "php": "^7.2",
+        "symfony-cmf/routing": "dev-master",
```
Comment (dbu) [inline composer.json:19]: note to self: when we see this branch build, lets tag the routing component and then adjust this version

**Comment by acrobat** [comment]
@dbu I've required the needed dev testing packages but I also needed a fix in the testing package, see symfony-cmf/testing#202.

I've temporarily required my fork, but due to github's ratelimiting the build currently fails. I will restart it in an hour or so.

File: composer.json [hunk 2]
```diff
@@ -56,5 +62,11 @@
         "branch-alias": {
             "dev-master": "2.x-dev"
         }
-    }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:acrobat/Testing.git"
```
Comment (alexander-schranz) [inline composer.json:69]: @acrobat travis have mostly problems with forks as you see the CI fails to clone the dependency. You can work around this problems by adding the following to this composer.json:

```json
"config" {
    "github-protocols": ["https"]
}
```

So we can see if the tests run successfully.

Comment (dbu) [reply composer.json:69]: i merged https://github.com/symfony-cmf/Testing/pull/202 now, using latest master of the original testing repo should pick that up now

**Comment by dbu** [comment]
now it installs again \o/

some symfony project setup seems to be missing now for the functional tests. (i think there has been quite some cleanup in symfony-cmf/testing because it was doing too much, and this bundle has not been updated accordingly)

**Comment by acrobat** [comment]
yes indeed, I was looking into the failures but I didn't understand why it would fail after my changes. But I will check what the cleanup was and what is missing here!

**Comment by acrobat** [comment]
@dbu It was actually a stupid mistake from my side with the fixes on the testing package.. The build should pass again when symfony-cmf/Testing#203 is merged!

**Comment by dbu** [comment]
i restarted the build

**Comment by acrobat** [comment]
Thanks @dbu, looks like everything is green now!

**Comment by dbu** [comment]
hooray! thanks a ton! i will do the tagging now

**Comment by alexander-schranz** [comment]
@acrobat @dbu Thats great News 🎉 🎂 . Also a big thanks from the whole @sulu team to both of you 👍

**Pull request merged**